decidim-core/app/jobs/decidim/machine_translation_fields_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  class MachineTranslationFieldsJob < ApplicationJob
+    queue_as :default
+
+    def perform(resource, field_name, field_value, locale, source_locale)
```

Comment:
Maybe at some docs? Like "Translates a field or attribute from a given resource" and document the params too

Reply:
Done!